nltools/data/brain_data.py

```diff
@@ -335,16 +335,54 @@ def shape(self):
 
         return self.data.shape
 
-    def mean(self):
-        """ Get mean of each voxel across images. """
+    def mean(self, axis=0):
+        ''' Get mean of each voxel or image
+            
+            Args:
+                axis: (int) across images=0 (default), within images=1
+            
+            Returns:
+                out: (float/np.array/Brain_Data)
+
+        '''
 
         out = deepcopy(self)
-        if len(self.shape()) > 1:
-            out.data = np.mean(self.data, axis=0)
-            out.X = pd.DataFrame()
-            out.Y = pd.DataFrame()
-        else:
+        if check_brain_data_is_single(self):
```

Comment:
Looks like you're missing an import for `check_brain_data_is_single` at the top of the `brain_data.py` file

Reply:
yeah, a bunch of these small things are getting fixed as I work through getting all of the tests to pass. this one is already fixed.